sklbench/utils/common.py

```diff
@@ -120,7 +120,7 @@ def flatten_list(input_list: List, ensure_type_homogeneity: bool = False) -> Lis
 
 
 def get_module_members(
-    module_names_chain: Union[List, str]
+    module_names_chain: Union[List, str],

```

Comment:
```suggestion
    module_names_chain: Union[List, str]
```
just cleaning up diff